vtex-apps/powerreviews
# Load product schema on product pages only

**What problem is this solving?**

As per google's documentation (https://developers.google.com/search/docs/advanced/structured-data/product#product) the Product schema should only be present on pages that describe a single product, therefor limiting this to store.product is mandatory in order to avoid errors 

**How should this be manually tested?**

Simply link the change here on a workspace where products have powerreviews on a search page as well as product page and inspect the source to check that the Product schema is only present on the product page.

**Screenshots or example usage:**

N/A - nothing to show.

PS: The reason why there are so many files changed is that I've run vtex setup, but in reality I've only changed RatingSummary.tsx and the typings for render-runtime.
